lib/rules/capitalized-comments.js

```diff
@@ -0,0 +1,258 @@
+                                                items: {
+                                                    type: "string"
+                                                },
+                                                uniqueItems: true
+                                            },
+                                            ignoreInlineComments: {
+                                                type: "boolean"
+                                            }
+                                        },
+                                        additionalProperties: false
+                                    },
+                                    block: {
+                                        type: "object",
+                                        properties: {
+                                            capitalize: { enum: ["always", "never"] },
+                                            ignorePattern: {
+                                                type: "string"
+                                            },
+                                            markers: {
+                                                type: "array",
+                                                items: {
+                                                    type: "string"
+                                                },
+                                                uniqueItems: true
+                                            },
+                                            ignoreInlineComments: {
+                                                type: "boolean"
+                                            }
+                                        },
+                                        additionalProperties: false
+                                    }
+                                },
+                                additionalProperties: false
+                            }
+                        },
+                        additionalProperties: false
+                    }
+                ]
+            }
+        ]
+    },
+
+    create(context) {
+
+        const normalizedOptions = getAllNormalizedOptions(context.options[0]),
+            sourceCode = context.getSourceCode();
+
+        //----------------------------------------------------------------------
+        // Helpers
+        //----------------------------------------------------------------------
+
+        /**
+         * Checks whether a comment is an inline comment.
+         *
+         * @param {ASTNode} comment The comment node to check.
+         * @returns {boolean} True if the comment is an inline comment, false
+         * otherwise.
+         */
+        function isInlineComment(comment) {
+            const firstToken = sourceCode.getFirstToken(comment),
+                previousToken = sourceCode.getTokenBefore(firstToken);
+
+            return previousToken &&
+                firstToken.loc.line === previousToken.loc.line;
```

Comment:
Though nit-picking, `previousToken` is not a `boolean` value, but JSDoc says `@returns {boolean}`.